Pspcl/Pspcl.Web/Views/Dashboard/Index.cshtml

```diff
@@ -0,0 +1,3 @@
+﻿@{
+    Layout = "~/Views/Shared/LeftSideBarLayout.cshtml";
```

Comment:
Wrong! It should be a partial view, HTML, HEAD, BODY tags should be under main layout view